ml-api-adapter/chart-handler-notification/configs/default.json

```diff
@@ -0,0 +1,118 @@
+{
+    "PORT": {{ .Values.service.externalPort }},
+    "ENDPOINT_SOURCE_URL": "http://{{ printf "%s-%s" .Release.Name .Values.config.central_services_host }}:{{ .Values.config.central_services_port }}/participants/{{"{{"}}fsp{{"}}"}}/endpoints",
+    "ENDPOINT_CACHE_CONFIG": {
+        "expiresIn": 180000,
+        "generateTimeout": 30000
+    },
+    "HOSTNAME": "{{ .Values.ingress.hosts.api }}",
+    "AMOUNT": {
+        "PRECISION": 10,
+        "SCALE": 2
+    },
+    "HANDLERS": {
+        "DISABLED": false,
+        "API": {
+            "DISABLED": false
+        }
+    },
+    "KAFKA": {
+        "TOPIC_TEMPLATES": {
+            "PARTICIPANT_TOPIC_TEMPLATE": {
+                "TEMPLATE": "topic-{{"{{"}}participantName{{"}}"}}-{{"{{"}}functionality{{"}}"}}-{{"{{"}}action{{"}}"}}",
+                "REGEX": "topic-(.*)-(.*)-(.*)"
+            },
+            "NOTIFICATION_TOPIC_TEMPLATE": {
+                "TEMPLATE": "topic-notification-event",
+                "REGEX": "topic-notification-event"
+            },
+            "FULFIL_TOPIC_TEMPLATE": {
+                "TEMPLATE": "topic-transfer-fulfil",
+                "REGEX": "topic-transfer-fulfil"
+            }
+        },
+        "CONSUMER": {
+            "NOTIFICATION": {
+                "EVENT": {
+                    "config": {
+                        "options": {
+                            "mode": 2,
+                            "batchSize": 1,
+                            "recursiveTimeout": 100,
+                            "messageCharset": "utf8",
+                            "messageAsJSON": true,
+                            "sync": true,
+                            "consumeTimeout": 1000
+                        },
+                        "rdkafkaConf": {
+                            "client.id": "ml-con-notification-event",
+                            "group.id": "kafka-ml-api-adapter",
+                            "metadata.broker.list": "{{ ( .Values.config.kafka_host | replace "$release_name" .Release.Name ) }}:{{ .Values.config.kafka_port }}",
```

Comment:
Any reason this doesn't follow the same pattern used by the other configs? aka define a var at the top of the file which is used here?